examples/assembly/Add_Array_(AArch64).asm

```diff
@@ -0,0 +1,191 @@
+# This file demonstrates adding 2 arrays using serial, SVE and NEON instructions.
```

Comment:
Is "serial" the word or is it "scalar"?

(genuine question, I'd call it scalar because it uses scalar registers, but I don't know what vectorisers call it)

Reply:
Serial does make sense, one at a time, so if that's also a common term then fine leave it as that.